app/src/main/java/de/westnordost/streetcomplete/screens/user/profile/LaurelWreath.kt

```diff
@@ -0,0 +1,80 @@
+package de.westnordost.streetcomplete.screens.user.profile
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.center
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.graphics.drawscope.inset
+import androidx.compose.ui.graphics.drawscope.withTransform
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.ui.theme.LeafGreen
+
+@Composable
+fun LaurelWreath(
```

Comment:
@matkoniecz I re-did the laurel wreath in compose. Feel free to look through the code to understand what I changed.